R/api.R

```diff
@@ -0,0 +1,136 @@
+##' Run a pkgbuilder server
+##'
+##' @title Run a pkgbuilder server
+##'
+##' @param versions A vector of 2-digit version numbers corresponding
+##'   to the versions of R to build packages for.  Workers will need
```

Comment:
Any reason not to start workers here to match the versions?

Reply:
The details of how a different system will store different versions of R alongside each other is platform-specific so it's not generally possible to do. Plus this way we could add or remove workers as we go.

What I would prefer to do is to have the server advertise no version at all, but let adding a worker make a version available. But to do that we need to be able to detect when workers join or leave and we can't do that on liteq